infrastructure/src/main/java/org/corfudb/infrastructure/log/StreamLog.java

```diff
@@ -50,9 +52,24 @@
     void compact();
 
     /**
-     * Get the global tail and stream tails.
+     * Get the global log tail.
      */
-    TailsResponse getTails();
+    Long getLogTail();
+
+    /**
+     * Get global and stream tails.
+     */
+    TailsResponse getAllTails();
```

Comment:
using a `response` class in a stream level doesn't seem like a good idea

Reply:
I agree. I've modified this accordingly.